lib/rspec/core/example.rb

```diff
@@ -577,7 +582,9 @@ class ExecutionResult
         #   this indicates whether or not it now passes.
         attr_accessor :pending_fixed
 
-        alias pending_fixed? pending_fixed
+        def pending_fixed?
+          !!pending_fixed
+        end
```

Comment:
This one can be set to a truthy value manually, but should also always be a boolean...